make/common.mk

```diff
@@ -75,7 +75,7 @@ integration: bin/$(BINNAME)
 #########################################
 
 fmt:
-	$Q gofmt -l -w $(SRC)
+	$Q goimports -local github.com/golangci/golangci-lint -l -w $(SRC)
```

Comment:
Should we add `goimports` to `make bootstrap` or just assume it's there because it's installed by most of the Go editor modes?

Reply:
Let's just assume it's there. If we hear about issues we'll add to `bootstrap`.